lib/jekyll-feed/feed.xml

```diff
@@ -115,7 +115,16 @@
         <summary type="html"><![CDATA[{{ post_summary | strip_html | normalize_whitespace }}]]></summary>
       {% endif %}
 
-      {% assign post_image = post.image.path | default: post.image %}
+      {% assign thumb_key = site.data.thumb_key %}
```

Comment:
We use the term "image" previously, not "thumb". Could you update this to "image_path_key" or "image_path_frontmatter_key"?

Reply:
OK, changing all ocurrences of "thumb" to "image" for consistency.